pom.xml

```diff
@@ -485,6 +485,71 @@
                   file="${project.basedir}/node_modules/papaparse/papaparse.min.js"
                   tofile="src/main/webapp/generated/papaparse.min.js"
                 />
+                <copy todir="src/main/webapp/generated/katex">
+                  <fileset dir="${project.basedir}/node_modules/katex/dist" />
+                </copy>
```

Comment:
Copy all of the katex/dist folder because it contains fonts referenced in the CSS